client/landing/stepper/utils/get-flow-from-url.ts

```diff
@@ -1,9 +1,12 @@
+import { ONBOARDING_FLOW } from '@automattic/onboarding';
 import { matchPath } from 'react-router';
 
-export const getFlowFromURL = (
-	pathname = window.location.pathname ?? '',
-	search = window.location.search
-) => {
+export const DEFAULT_FLOW = ONBOARDING_FLOW;
+
+export const getFlowFromURL = ( pathname?: string, search?: string ) => {
+	pathname ??= typeof window !== 'undefined' ? window.location.pathname ?? '' : '';
```

Comment:
Took me a minute to break this down 😆 

But I see: we want to only define pathname if it's not defined already, and we'll set it as `window.location.pathname` otherwise (unless that's also not defined and if we do have the window object).

Reply:
Yeah, it's a ridiculous number of `?`s 🙈 I feel like some explicit parenthesis would make this easier to read. But Prettier insists on removing them.